gcpkms/keysource.go

```diff
@@ -249,7 +261,23 @@ func getGoogleCredentials() ([]byte, error) {
 		if _, err := os.Stat(defaultCredentials); err == nil {
 			return os.ReadFile(defaultCredentials)
 		}
+
 		return []byte(defaultCredentials), nil
 	}
-	return nil, nil
+	return nil, fmt.Errorf("could not find Google credential file")
```

Comment:
Can you update the function doc comment to reflect behavior change? Right now, it incorrectly reads `and may return a nil byte slice if no value is set.`

Reply:
I also changed the description of the SopsGoogleCredentialsOAuthToken, where i forgot to update after removing the possibility of using a file path! 

Thank you for all of your comments, and time! 🌻